authc-model/src/main/java/com/symphony/s2/authc/model/ISupportContext.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright 2020 Symphony Communication Services, LLC.
+ *
+ * All Rights Reserved
+ */
+
+package com.symphony.s2.authc.model;
+
+/**
+ * 
+ * An authentication and authorization context for some process.
+ * 
+ * An instance of this type is passed to service methods to indicate the identity and permissions of the service caller.
+ * 
+ * @author Geremia Longobardo
+ *
+ */
+public interface ISupportContext
+{
+
+  /**
+   * 
+   * @return true if the Support Portral user is enabled to perform an action
+   */
+  boolean hasEntitlements();
```

Comment:
Surely this needs a parameter?

Reply:
the concrete class has an arrayList, the implementation should be if(list.contains(entitlement)) return true